src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+.header {
+  background-size: cover;
+  background-position: center;
+  background-image: url(../images/header.png);
+
+  @include onTablet() {
+    background-image: url(../images/header-L.png);
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    box-sizing: border-box;
+    height: 100vh;
```

Comment:
If I'm not mistaken the header height should be as in the Figma layout and only desktop size 100vh?
this is not a mistake just note.